src/Core/Path.php

```diff
@@ -9,7 +9,7 @@
  */
 class Path
 {
-    const TRIM_CHARS = ' /\\';
+    public const TRIM_CHARS = ' /\\';
```

Comment:
```suggestion
    const TRIM_CHARS = ' /\\';
```
Even though `public` is the default for a const, we don't want any signature changes outside of a new major version (CMS 6)

Reply:
Oops I don't even remember making this change.. Maybe my IDE took over!